### ISSUE TITLE
About training details

### ISSUE THREAD

**Issue by fuenwang** [SensorsINI/DHP19#20]
Hi, thanks for contributing this great work and dataset!
I would like to implement a pytorch version and some training details seems missing. Could you provide me the details like 
  1. batch size
  2. the parameter of RMSProp (momentum, weight_decay, .....)
  3. Data augmentation
  4. Model initialization (xavier or uniform....)
  5. Is your conv/deconv layer including batchnorm layer?
  6. In your python code, you define " decay_heatmap(heatmap, sigma2=4):", in which sigma is 4, but the sigma in the paper is 2 instead. Which one is the final value you use for training?

Thanks very much!
Bests.
